.github/workflows/merge.yml

```diff
@@ -118,3 +120,23 @@ jobs:
             *.cache-to=type=gha,scope=bin-image,mode=max
             *.attest=type=sbom
             *.attest=type=provenance,mode=max,builder-id=https://github.com/${{ env.GITHUB_REPOSITORY }}/actions/runs/${{ env.GITHUB_RUN_ID }}
+
+  desktop-edge-test:
+    runs-on: ubuntu-latest
+    needs: bin-image
+    steps:
+      -
+        name: Trigger Docker Desktop e2e with edge version
+        uses: actions/github-script@v6
+        with:
+          github-token: ${{ secrets.DOCKERPUBLICBOT_WRITE_PAT }}
```

Comment:
That’s a token for Docker Hub, isn’t it? I’m not sure it’s going to help there.

Reply:
Yep, that's why this PR is still a draft, I also need to update the way I'll get the image tag to pass to the workflow